src/components/civ-pool-button-bar/civ-pool-button-bar.scss

```diff
@@ -44,6 +44,7 @@
   }
 
   .save-pool-button,
+  .private-toggle-button,
```

Comment:
I feel like there could be a more generic button classname at this point, but no big deal

Reply:
I like to keep them separate in case they need special attention in the future.